packages/@lwc/integration-karma/test/light-dom/root/index.spec.js

```diff
@@ -93,4 +93,13 @@ describe('root light element', () => {
             [window, nodes.button, composedPath],
         ]);
     });
+
+    it('querySelector should return slotted elements', () => {
+        setFeatureFlagForTest('ENABLE_LIGHT_GET_ROOT_NODE_PATCH', true);
```

Comment:
I believe this `setFeatureFlagForTest('ENABLE_LIGHT_GET_ROOT_NODE_PATCH', true);` should be in a `beforeEach` so that it can be unset in an `afterEach`.